src/routes/SelectWorkType/index.jsx

```diff
@@ -94,7 +95,12 @@ const SelectWorkType = ({
       challenge?.id,
       challenge?.legacy?.selfService
     );
-
+  
+  const breadcrumb = [
+    {url:'/self-service/dashboard', name:"my work"},
```

Comment:
let's use sentence casing for the names, even though they are transformed to all caps...
`name: "My work"`